siad/renter.go

```diff
@@ -33,6 +41,22 @@ func (d *daemon) renterDownloadHandler(w http.ResponseWriter, req *http.Request)
 	writeSuccess(w)
 }
 
+// renterDownloadqueueHandler handles the API call to request the download
+// queue.
+func (d *daemon) renterDownloadqueueHandler(w http.ResponseWriter, req *http.Request) {
+	downloads := d.renter.DownloadQueue()
+	downloadSet := make([]DownloadInfo, 0, len(downloads))
+	for _, dl := range downloads {
+		downloadSet = append(downloadSet, DownloadInfo{
+			Completed:   dl.Completed(),
+			Destination: dl.Destination(),
+			Nickname:    dl.Nickname(),
+		})
```

Comment:
Like I said, I think they'll be more useful once we want to do more sophisticated things with them. Right now they're just needlessly redundant.
I guess there's no harm in implementing them now though.